src/modules/gifmaker/GifRecorderOverlay/MainPage.xaml.cs

```diff
@@ -40,12 +41,14 @@ private void CaptureButton_Click(object sender, RoutedEventArgs e)
         }
         private void RecordPauseButton_Click(object sender, RoutedEventArgs e)
         {
+            Window.Current.CoreWindow.PointerCursor = new Windows.UI.Core.CoreCursor(Windows.UI.Core.CoreCursorType.Arrow, 1); 
```

Comment:
What is 1 for, something that we can do is either set a constant variable that basically means 1 and put it there OR do like foo( 1 /* the 1st index */ ) . *using the comment to indicate what the 1 means